android/tangram/src/com/mapzen/tangram/MapController.java

```diff
@@ -650,6 +663,26 @@ public void pickFeature(float posX, float posY) {
             nativePickFeature(mapPointer, posX, posY, featurePickListener);
         }
     }
+    /**
+     * Set a listener for label pick events
+     * @param listener Listener to call
+     */
+    public void setLabelPickListener(LabelPickListener listener) {
+        labelPickListener = listener;
+    }
+
+    /**
+     * Query the map for labeled features at the given screen coordinates; results will be returned
+     * in a callback to the object set by {@link #setLabelPickListener(LabelPickListener)}
+     * @param posX The horizontal screen coordinate
+     * @param posY The vertical screen coordinate
+     */
+    public void pickLabels(float posX, float posY) {
```

Comment:
This one is necessary the same way `pickFeatureAt` is used, and can be used on a callback of a single tap if you want, but not only.

In the demo app it's called on a `singleTapConfirmed` https://github.com/tangrams/tangram-es/blob/feature-selection-label-infos/android/demo/src/com/mapzen/tangram/android/MainActivity.java#L136